core/http/endpoints/openai/files.go

```diff
@@ -120,6 +122,7 @@ func getFileFromRequest(c *fiber.Ctx) (*schema.File, error) {
 // @Router /v1/files/{file_id} [get]
 func GetFilesEndpoint(cm *config.BackendConfigLoader, appConfig *config.ApplicationConfig) func(c *fiber.Ctx) error {
 	return func(c *fiber.Ctx) error {
+		c.Set("LocalAI-Machine-Tag", appConfig.MachineTag)
```

Comment:
this could be quite sensitive to add to each reply without the user to explicitly set it out. maybe it should be behind a feature flag in the config

Reply:
yeah, i'll create that opt-in as well